src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3878,7 +3878,7 @@ private static SyntaxToken GetImplicitConstructorBodyToken(CSharpSyntaxNode cont
         {
             NamedTypeSymbol containingType = constructor.ContainingType;
             NamedTypeSymbol baseType = containingType.BaseTypeNoUseSiteDiagnostics;
-            Location diagnosticsLocation = constructor.Locations.FirstOrNone();
+            Location diagnosticsLocation = constructor.GetFirstLocationOrNone();
```

Comment:
this commit should be the mechanical change of Locations.FirstOrNont() to GetFirstLocationOrNone() (A new helper i added in this commit)